products/error_tracking/backend/tools/search_issues.py

```diff
@@ -0,0 +1,248 @@
+        - event.$os: Operating system
+        - event.$device_type: Device type (Desktop, Mobile, Tablet)
+        - event.$current_url: URL where error occurred
+        - event.$lib: SDK/library used (web, posthog-python, posthog-node, etc.)
+
+        ## filterTestAccounts (optional)
+        - true: Exclude internal/test accounts
+        - false: Include all accounts
+
+        ## volumeResolution (REQUIRED)
+        Resolution for volume chart data. Use 1 for daily buckets.
+
+        {SEARCH_QUERY_EXAMPLES}
+        """).strip()
+    )
+
+
+class SearchErrorTrackingIssuesTool(MaxTool):
+    name: Literal["search_error_tracking_issues"] = "search_error_tracking_issues"
+    args_schema: type[BaseModel] = SearchErrorTrackingIssuesArgs
+    description: str = dedent("""
+        Search for error tracking issues based on criteria like status, search text, date range, and ordering.
+
+        # When to use this tool:
+        - User asks to find, search, or list error tracking issues
+        - User asks about errors, exceptions, or issues in their application
+        - User wants to filter errors by status, date, frequency, or other criteria
+        - User asks questions like "show me recent errors" or "what are the most common issues"
+
+        # What this tool returns:
+        A formatted list of matching issues with their name, status, occurrence count, and other key metrics.
+        """).strip()
+
+    async def _arun_impl(self, query: ErrorTrackingQuery) -> tuple[str, ToolMessagesArtifact | None]:
+        # Adding some sane default limits
+        if query.limit is None or query.limit <= 0:
+            query.limit = 25
+        elif query.limit > 100:
+            query.limit = 100
+
+        # Default to active issues if no status specified
+        if query.status is None:
+            query.status = Status2.ACTIVE
+
+        try:
+            utc_now = timezone.now().astimezone(UTC)
+            executor = AssistantQueryExecutor(self._team, utc_now)
+            query_results = await executor.aexecute_query(query)
+        except Exception as e:
+            capture_exception(e)
+            logger.exception("Error executing error tracking query", error=str(e))
+            return f"Error searching for issues: {e}", None
+
+        return self._format_results(query_results), None
+
+    def _format_results(self, response: dict) -> str:
+        """Format query results as text output."""
+        results = response.get("results", [])
+
+        if not results:
+            return "No issues found matching your criteria."
+
+        total_count = len(results)
+        if total_count == 1:
+            content = "Found 1 issue matching your criteria:\n\n"
+        else:
+            content = f"Found {total_count} issues matching your criteria:\n\n"
+
+        # Show up to 10 issues in the response
+        for i, issue in enumerate(results[:10], 1):
+            content += self._format_issue(i, issue)
```

Comment:
Let's add typings here:

```suggestion
            content += self._format_issue(i, ErrorTrackingIssue.model_validate(issue))
```